teammapper-backend/src/main.ts

```diff
@@ -49,6 +50,26 @@ async function bootstrap() {
     })
   )
 
+  // Serve static assets first (these take priority)
+  app.useStaticAssets(join(__dirname, '..', 'client/browser/assets'), {
+    prefix: '/assets/',
+    setHeaders: (res, path) => {
+      // Cache static assets
+      if (
+        path.match(
+          /\.(js|css|png|jpg|jpeg|gif|ico|svg|woff|woff2|ttf|eot|map)$/
```

Comment:
js seems to appear twice, here and in the other rule?

Reply:
or maybe I am miss interpreting. So all requests to /assets will be served from here and take priority. All other requests will be redirected to the frontend app.

Still a few things where I would be careful: 
- The cache time seems very high (1 year). What happens if we update the content of a js file? I think the chunks change their file names, not so sure about the static assets. This sounds too high. I would propose a time window of a day. And then force a revalidation.